api/src/opentrons/protocol_api/labware.py

```diff
@@ -812,10 +836,23 @@ def return_tips(self, start_well: Well, num_channels: int = 1) -> None:
         :type start_well: :py:class:`.Well`
         :param num_channels: The number of channels for the current pipette
         :type num_channels: int
+
+        .. deprecated:: 2.14
+            Modification of tip tracking state outside :py:meth:`.reset` has been deprecated.
         """
+        if self._api_version >= ENGINE_CORE_API_VERSION:
+            raise APIVersionError(
+                "Labware.previous_tip has been deprecated."
+                " To modify tip state, use Labware.reset"
+            )
```

Comment:
"Deprecated" -> "unsupported", plus "previous_tip" -> "return_tips()."

```suggestion
            raise APIVersionError(
                "Labware.return_tips() is unsupported in this API version."
                " Use Labware.reset() instead."
            )
```